ibc/host/identifiers.go

```diff
@@ -0,0 +1,100 @@
+package host
+
+import (
+	"fmt"
+	"math/rand"
+	"strings"
+	"time"
+
+	coreTypes "github.com/pokt-network/pocket/shared/core/types"
+)
+
+const (
+	defaultIdentifierLength = 32
```

Comment:
1. Identifier for what? `defaultXXXIDLength`
2. What's the unit? Chars? Bits?

Rename :)